### ISSUE TITLE
low performance or not working at all

### ISSUE THREAD

**Issue by adserba** [fireice-uk/xmr-stak-amd#103]
hi! I am trying to launch xmr-stak-amd on a firepro v7900. The only config that I coud get running is:
     { "index" : 0, "intensity" : 256, "worksize" : 8, "affine_to_cpu" : false },   --86h/s
it also works with worksize 16 and intensity 128 but gives 24h/s.
any suggestion?
I am asking because with claymore 9.7  the same card does 295 h/s
thnk you

**Comment by psychocrypt** [comment]
You need to increase the worksize to 756 or 1024 to get better performance.

**Comment by adserba** [comment]
thank you for your answer,
if I try
{ "index" : 0, "intensity" :756, "worksize" : 8, "affine_to_cpu" : false } 
it returns
[2017-09-15 05:24:37] : Compiling code and initializing GPUs. This will take a while...
[2017-09-15 05:24:37] : Device 0 work size 8 / 256.
[2017-09-15 05:24:37] : Error CL_INVALID_BUFFER_SIZE when calling clCreateBuffer to create hash scratchpads buffer.
same for 1024...
if it helps here is clinfo 
[clinfo.txt](https://github.com/fireice-uk/xmr-stak-amd/files/1305265/clinfo.txt)

**Comment by adserba** [comment]
If I try to increase worksize , as you said, it doesn't work as worksize needs to be 1,8...so  I changed the intensity ... no ideeas?

**Comment by psychocrypt** [comment]
Yes intensity is the value I mean. I mixed up the names from nvidia and the
amd miner.

Am 17.09.2017 08:48 schrieb "adserba" <notifications@github.com>:

**Comment by adserba** [comment]
anyting above 256 gives me Error CL_INVALID_BUFFER_SIZE when calling clCreateBuffer to create hash scratchpads buffer. --- i tried increasing steb by step...it doesn't work thath's why I attached the clinfo file...maybe that helps. the card is opencl 1.2...

**Comment by dnsBlah** [comment]
I also have poor performance, but I'm kinda newbie...
I have 2 GPU's
Using ethminer I get 10M/s and using he xmr-stak-amd I only get around 700H/s
```
AMD FirePro D700:

      Chipset Model: AMD FirePro D700
      Type: GPU
      Bus: PCIe
      Slot: Slot-1
      PCIe Lane Width: x16
      VRAM (Total): 6144 MB
      Vendor: ATI (0x1002)
      Device ID: 0x6798
      Revision ID: 0x0000
      ROM Revision: 113-C3861J-687
      Automatic Graphics Switching: Supported
      gMux Version: 4.0.11 [3.2.8]
      EFI Driver Version: 01.0D.687
      Metal: Supported
      Displays:
        Display:
          Resolution: 1280 x 1024 @ 60 Hz
          Pixel Depth: 32-Bit Color (ARGB8888)
          Main Display: Yes
          Automatically Adjust Brightness: Yes

    AMD FirePro D700:

      Chipset Model: AMD FirePro D700
      Type: GPU
      Bus: PCIe
      Slot: Slot-2
      PCIe Lane Width: x16
      VRAM (Total): 6144 MB
      Vendor: ATI (0x1002)
      Device ID: 0x6798
      Revision ID: 0x0000
      ROM Revision: 113-C3861J-687
      Automatic Graphics Switching: Supported
      gMux Version: 4.0.11 [3.2.8]
      EFI Driver Version: 01.0D.687
      Metal: Supported
      Displays:
        Display:
          Resolution: 1280 x 1024 @ 60 Hz
          Pixel Depth: 32-Bit Color (ARGB8888)
          Main Display: Yes
          Automatically Adjust Brightness: Yes
```

```
"gpu_threads_conf" : [
        { "index" : 0, "intensity" : 2000, "worksize" : 8, "affine_to_cpu" : false },
        { "index" : 1, "intensity" : 2000, "worksize" : 8, "affine_to_cpu" : false },
],
```

**Comment by psychocrypt** [comment]
Do you get 700 h/s per gpu? The hash algorithm for eth and xmr differ.
700h/s for one device is a very good result.

Am 18.09.2017 12:43 Nachm. schrieb "dnsBlah" <notifications@github.com>:

**Comment by dnsBlah** [comment]
Hi Pyschocrypt,

thanks for your reply!
No it's the total..

But can that be correct? 10M/s for ETH to 700H.s for XMR?

**Comment by msilvoli** [comment]
@adserba
Did you manage the problem?
I have a V5800 and I get the same error. In fact I was trying to use a Linux but I couldn't configure the video card on it so I'm using the Windows Server 2012. With windows I am using the AMD drivers but I can't put anything higher than 180 on Worksize because the drivers crash.
Max hash is 85H/s.

**Comment by adserba** [comment]
no...nothing above 80h/s....you can try claymore....it gave me about 300h/s

**Comment by msilvoli** [comment]
I'll check it out !! Thank you !
